### ISSUE TITLE
[Improvement] 'Daily Plan' | tab 'Outstanding'

### ISSUE THREAD

**Issue by KostyaEver** [ever-co/ever-teams#2618]
Clarify with @evereq 
tab 'Outstanding' should contain collapsed plans and display all tasks that were not completed

The only concern if it should contain tasks split per Plans?

As I see it.
There should be a list of all tasks that are from all previous plans and were NOT completed (with statuses TODO, In progress, Blocked)

In that point, it is not important for users from which plan tasks are, the more important is to see the uncompleted tasks to be able to add them to new plans and complete them.

How it works
Every day at 0:00 am the system should check 'Daily Plan' of the previous day (if there was not any plan, no need to check)
All tasks in that 'Plan' that have uncompleted statuses (TODO, In progress, Blocked) should be added to the tab 'Outstanding' automatically.

**Comment by evereq** [comment]
We may not need any system like you describe that do something at 00:00 am. Essentially this feature is about correctly filtering of data and display it as nothing may get "moved" actually between "Plans", more like tasks just have a field "Planned Date" and so that defines to which "Plan" that task belong and when we want to show all outstanding tasks, we basically show all tasks with planned date > today date and not completed yet. Same as if we want to show tasks that planned for tomorrow, we only show tasks where PlannedDate = Tomorrow Date and so on. 
I am not sure if we worked on that already or not and how we decided to implements Plans functionality. One way was to create Plans tables and another is just go with that "PlannedDate" approach. We need to analyze all those tasks you created and see if it's doable without introduction of Plans entity and just with "PlannedDate" field!

**Comment by KostyaEver** [comment]
We do need '0:00' feature because all tasks that are considered as uncompleted (the system should know it from statuses) move from a plan that from 'Today' has become 'Yesterday' at 0:01 am. It should add/display all of them to 'Outstanding' 

Also here is the moment about why we need attributes for statuses, since the system should recognize all tasks that are NOT (done, completed, ready for review etc)


The question in the beginning was more about which VIEW is better to use, but we will do both) I already described in separate tasks
@evereq

**Comment by GloireMutaliko21** [comment]
@KostyaEver Since we have this closed ticket https://github.com/ever-co/ever-teams/issues/2626, see PR https://github.com/ever-co/ever-teams/pull/2687, Is there some additional requirement should be implemented here ???

CC: @evereq

**Comment by KostyaEver** [comment]
@GloireMutaliko21 
Looks like yes, just to be sure that logic how tasks with statuses get into this tab.
I hope that it is not hardcoded by status name, because user can easily rename status from 'TODO' - 'Completed' that we will see in Outstanding a lot of tasks that 'Completed' but in reality they are 'TODO' (anyway it is part of different task)

**Comment by GloireMutaliko21** [comment]
You are right @KostyaEver ! Actually, think they are hard coded with status name and think modifying the status name of tasks should delete them from outstanding tab....

**Comment by KostyaEver** [comment]
@GloireMutaliko21 

I have a solution for it we can add 'type' attributes for each status when we create it so that it will resolve the issue.

Types can be the same as standard Status Names (Non Active, In progress,  QA, Done) etc

The name can be any, so the system will rely on type attributes with no relation to Custom Names. (Relying on Custome Names is a high risk in terms of status automation functionality)

But we should get confirmation from @evereq

**Comment by GloireMutaliko21** [comment]
Great @KostyaEver ! Waiting for @evereq then I can go over this ticket
